### ISSUE TITLE
No matching version found for @prisma/debug@0.0.0.

### ISSUE THREAD

**Issue by martineboh** [prisma/prisma#6425]
## Bug description
I am trying to install prisma migrate (npm i @prisma/migrate) since last night with no success. I keep getting the below messages:

```js
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @prisma/debug@0.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of '@prisma/migrate'
npm ERR! notarget
```

Prisma 2.20.1
Windows 10

I think this might be related to https://github.com/prisma/prisma/issues/6422#issue-850779191

**Comment by timsuchanek** [comment]
Thanks for reporting @martineboh ! We just released a @prisma/migrate@2.20.2 with [the correct dependency versions](https://unpkg.com/browse/@prisma/migrate@2.20.2/package.json).

**Comment by martineboh** [comment]
Oh! Thanks
